README.md

```diff
@@ -50,7 +60,17 @@ A convenience script has been provided which installs Chocolatey (package manage
 
 ### Build and Install
 
-Run `.\build-windows.ps1` in a new powershell terminal to build the source with support for all available database engines (default: SQL server) and install the project into a local directory (default: `install`) under the repository root. To compile with support for all database engines, run `.\build-windows.ps1 -SqlServer ON -MariaDB ON -Sqlite ON`. `-Debug` can be added to the command-line arguments to build in debug mode, otherwise, release mode is used.
+Run `.\build-windows.ps1` in a new powershell terminal to build the source with support for all available database engines (default: SQL server) and install the project into a local directory (default: `install`) under the repository root. To compile with support for all database engines, run `.\build-windows.ps1 -SqlServer ON -MariaDB ON -Sqlite ON` . `-Debug` can be added to the command-line arguments to build in debug mode, otherwise, release mode is used.
```

Comment:
Was there something wrong with the formatting that requires this change? If not, please remove the space so the diff isn't as noisy.